src/functionaltests/java/com/ericsson/ei/notifications/ttl/TestTTLSteps.java

```diff
@@ -0,0 +1,225 @@
+
+            // setting 1 second TTL on index in db
+            mongoDBHandler.createTTLIndex(missedNotificationDatabase, missedNotificationCollectionName,
+                    indexName, 1);
+            Boolean isInserted = mongoDBHandler
+                    .insertDocument(missedNotificationDatabase,
+                            missedNotificationCollectionName,
+                            missedNotificationDocument.toString());
+            assertEquals("Failed to create missed notification in database", true, isInserted);
+
+            List<String> result = mongoDBHandler.getAllDocuments(missedNotificationDatabase,
+                    missedNotificationCollectionName);
+            assertEquals("Database " + missedNotificationDatabase + " is not empty.", false, result.isEmpty());
+        }
+
+        @Given("^Aggregated object is created in database with index \"([A-Za-z0-9_]+)\"$")
+        public void aggregated_object_is_created_in_database(String indexName) throws IOException, ParseException {
+
+            JsonNode aggregatedObject = getJSONFromFile(AGGREGATED_OBJECT_FILE_PATH);
+            BasicDBObject aggregatedDocument = prepareDocumentWithIndex(aggregatedObject, indexName);
+
+            // setting 1 second TTL on index in db
+            mongoDBHandler.createTTLIndex(aggregationDataBaseName, aggregationCollectionName,
+                    indexName, 1);
+            Boolean isInserted = mongoDBHandler.insertDocument(aggregationDataBaseName,
+                    aggregationCollectionName, aggregatedDocument.toString());
+            assertEquals("Failed to create aggregated object in database",true, isInserted);
+
+            List<String> result = mongoDBHandler.getAllDocuments(aggregationDataBaseName,
+                    aggregationCollectionName);
+            assertEquals("Database " + aggregationDataBaseName + " is not empty.", false, result.isEmpty());
+        }
+
+        @When("^I sleep for \"([^\"]*)\" seconds")
+        public void i_sleep_for_time(int sleepTime) throws Throwable {
+            LOGGER.debug("Sleeping for " + sleepTime + " at " + LocalTime.now());
+            // The background task deleting notifications from Mongo DB
+            // runs every 60 seconds
+            TimeUnit.SECONDS.sleep(sleepTime);
+        }
+
+        @Then("^\"([^\"]*)\" has been deleted from \"([A-Za-z0-9_]+)\" database$")
+        public void has_been_deleted_from_database(String collection, String database) {
+            LOGGER.debug("Checking " + collection + " in " + database);
+            List<String> result = mongoDBHandler.getAllDocuments(database, collection);
+            assertEquals("Database is not empty.", true, result.isEmpty());
+        }
+
+        // SCENARIO @TestNotificationRetries
+
+        @Given("^Subscription is created$")
+        public void create_subscription_object() throws IOException, JSONException {
+            LOGGER.debug("Starting scenario @TestNotificationRetries.");
```

Comment:
What EI Logic are you testing here? Looks like you are testing your own code only.